spec/models/system_notification_spec.rb

```diff
@@ -69,4 +70,44 @@
       end
     end
   end
+
+  describe 'Methods' do
+    let(:notifcation_one) { create :system_notification }
+    let(:notifcation_two) { create :system_notification, title: 'Special one' }
+    before do
+      create :user_system_notification,
+             user: user,
+             system_notification: notifcation_one
+      create :user_system_notification,
+             user: user,
+             system_notification: notifcation_two
+    end
+
+    it 'get last notifications without search' do
+      result = SystemNotification.last_notifications(user)
+      expect(result.length).to eq 2
+      expect(result.first).to respond_to(
```

Comment:
This test doesn't make sense here I think

Reply:
It's more for test right result format. Cause if response structure will be changed, system notifications view may failed.